### DISCUSSION TITLE
Montage html coordinates incorrect when larger image is added

### DISCUSSION THREAD

**Discussion by robertcorponoi** [ImageMagick/ImageMagick#6166] [Help]
### ImageMagick version

7.1.1-2

### Operating system, version and so on

macOS Ventura 13.2.1

### Description

I want to create a montage from a set of images and use the generated html to get the position of each image in the montage. I have three images in the collage:

[![image-00.png][1]][1]
[![image-01.png][2]][2]
[![image-02.png][3]][3]

with their sizes being:

- `image-00.png` = `19 x 21`
- `image-01.png` = `20 x 22`
- `image-02.png` = `25 x 30`

When I create a montage with:

```sh
magick montage *.png -mode Concatenate -background transparent info.html  
``` 

using just the first two images, the html `coords` look correct:

```html
<map id="image" name="image">
  <area href="image-00.png" shape="rect" coords="0,0,19,21" alt="" />
  <area href="image-01.png" shape="rect" coords="20,0,39,21" alt="" />
</map>
```

However, when I add in the third image, the coordinates are different and incorrect as the first two frames are now larger:

```html
<map id="image" name="image">
  <area href="image-00.png" shape="rect" coords="0,0,24,29" alt="" />
  <area href="image-01.png" shape="rect" coords="25,0,49,29" alt="" />
  <area href="image-02.png" shape="rect" coords="50,0,74,29" alt="" />
</map>
```

I'm not sure why this happens since the dimensions of the first two frames didn't change. I'd like to be able to grab the exact position of each image from the html but I'm not sure how to do so when the coordinates aren't correct.

  [1]: https://i.stack.imgur.com/sEjz9.png
  [2]: https://i.stack.imgur.com/jbU1g.png
  [3]: https://i.stack.imgur.com/mmoDC.png

**Comment by urban-warrior** [comment]
Each image is a different size.  If you want precise coordinates, make them all the same size as in:

```
magick montage -geometry 25x25 image-0?.png -mode Concatenate -background transparent info.html
```
